### ISSUE TITLE
Can't set transparent background in settings UI, only colors that obscure text under cursor

### ISSUE THREAD

**Issue by ssteinerx** [olmokramer/atom-block-cursor#28]
So I had to go edit the settings by hand in ~/.atom/config.cson, but I'm not sure they're going to survive...

**Comment by olmokramer** [comment]
Yes this is a known issue. I did a request for a better color picker in the settings view some time ago, but nothing really happened... There are two ways to solve this problem:
- I (or someone else) can make a PR on the [settings-view](https://github.com/atom/settings-view) repo that replaces the color picker with a fancier one. I don't have the time to do this, though.
- I can use `object` config schemes for the colors instead of `color` schemes. I really don't like this option because it adds a _lot_ of fields to the settings page (12 extra for each cursor variant) and you can't specify colors with a CSS string when manually editing `config.cson`.

There is actually a third option: make a custom UI to edit the cursors. But again, I don't have time right now to do that. This is something I really want to do, though, as it would make editing cursors a much more pleasant experience :)

Editing your `config.cson` by hand should be safe. I believe you can use any valid CSS color string or an object with `red`, `green`, `blue` and `alpha` properties where the former 3 take a value between `0` and `255` and `alpha` is a value between `0` (transparent) and `1` (opaque).

**Comment by rinopo** [comment]
Hi,
The current interface was somewhat misguiding for me, too, since it results in an opaque cursor that hides the text.

How about replacing the current color picker buttons with simple text input boxes to accept CSS color expression?
